jax/third_party/numpy/linalg.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
This can be removed, JAX is Python 3 only :tada:

Reply:
Done!